sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuDataWritingCommandExec.scala

```diff
@@ -113,8 +113,17 @@ case class GpuDataWritingCommandExec(cmd: GpuDataWritingCommand, child: SparkPla
     extends ShimUnaryExecNode with GpuExec {
   override lazy val allMetrics: Map[String, GpuMetric] = GpuMetric.wrap(cmd.metrics)
 
-  private lazy val sideEffectResult: Seq[ColumnarBatch] =
-    cmd.runColumnar(sparkSession, child)
```

Comment:
GpuDataWritingCommand.runColumnar should be removed, or at least a followup to do so.